.github/workflows/verifyFreezePeriod.yml

```
@@ -13,8 +13,9 @@ jobs:
         run: |
           today=$(TZ=UTC date "+%Y%m%d")
           tomorrow=$(TZ=UTC date -d "+1 days" "+%Y%m%d")
-          calID="cHJmazI2ZmRtcHJ1MW1wdGxiMDZwMGpoNHNAZ3JvdXAuY2FsZW5kYXIuZ29vZ2xlLmNvbQ"
-          curl "https://calendar.google.com/calendar/u/0/htmlembed?src=${calID}&mode=AGENDA&ctz=UTC&dates=${today}/${tomorrow}" | grep -i -q -e "stabilization"
+          calURL="https://calendar.google.com/calendar/u/0/htmlembed?src=cHJmazI2ZmRtcHJ1MW1wdGxiMDZwMGpoNHNAZ3JvdXAuY2FsZW5kYXIuZ29vZ2xlLmNvbQ"
+          echo "Query calender ${calURL}"
```

I would fix a typo and possibly also rephrase this to:

```suggestion
          echo "Querying calendar ${calURL}"
```

I don't know local conventions though, so maybe `Query` is just fine as well...

While at it, there is another typo on a line above: `stabilitzation`.

---

Thank you. I applied it as suggested. Since I'm not a native speaker I'm not sure which one is better. Probably both is possible in the end.